internal/servegit/serve.go

```diff
@@ -170,7 +170,7 @@ func (s *Serve) Repos() ([]Repo, error) {
 		return nil, nil
 	}
 
-	err = filepath.Walk(root, func(path string, fi os.FileInfo, fileErr error) error {
+	err = filepath.WalkDir(root, func(path string, fi fs.DirEntry, fileErr error) error {
```

Comment:
Do you still need the `if !fi.IsDir()` below then?

Reply:
yes walkdir still goes over files.